apps/damap-frontend/src/themes/custom-theme.scss

```
@@ -122,10 +123,27 @@ html {
 
   /* Support for IE. */
   font-feature-settings: "liga";
+
```

```suggestion
  // Set default icons to filled
  font-variation-settings: "FILL" 1;
  // class to set icons outlined. Example usage: <mat-icon class="outline">delete</mat-icon>
  &.outline {
    font-variation-settings: "FILL" 0;
  }
```

To make the default icon set filled instead of outlined. See also css on the right side: https://fonts.google.com/icons?selected=Material+Symbols+Outlined:delete:FILL@1;wght@400;GRAD@0;opsz@24&icon.size=24&icon.color=%23000000&icon.style=Outlined&icon.query=delete

---

Yes, you can either add it like this, or as i removed in the tooltip the outlined part. Both works.